manifests/v1beta1/katib-controller/overlays/application/application.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: app.k8s.io/v1beta1
```

Comment:
Do we need to maintain these overlays ?

Reply:
For the application overlays, we are discussing about moving away from them soon:
https://github.com/kubeflow/manifests/issues/1715

What we could do is put them in an `installs/kubeflow` kustomization, since they are used when Katib is installed alongside other Kubeflow components.

Reply:
@yanniszark So should we add the application overlay for Katib now or it's better to remove all labels and add them later with an Application, once we need it.
Since currently, many Katib labels are incorrect for the Application: https://github.com/kubeflow/manifests/pull/1669.

Reply:
@andreyvelich in general, we are moving away from using the Application CR: https://github.com/kubeflow/manifests/issues/1715#issuecomment-779402779

For this PR, I would suggest leaving them as is. The initial plan is to just get the manifests under the control of the upstream repo (katib) and then we can make changes incrementally.